.github/workflows/release.yml

```diff
@@ -121,3 +121,50 @@ jobs:
         env:
           SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
           SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+  merge_pr_when_e2e_success:
```

Comment:
I think It would be simpler overall to not include this job in this release workflow.
For example, it is better to create a workflow such as `pr-auto-merge.yaml` and implement it within that workflow to reduce the number of job conditions, etc.

Reply:
thank you for your comment :) I have separated the workflow as suggested :)